front-end/src/app/components/home/home.component.html

```diff
@@ -5,22 +5,22 @@ <h1 class="ui inverted header">
       osuStuff Play
     </h1>
 
-    <h4>
+    <h3>
       A new fun way to play casual osu matches and tournaments.
-    </h4>
+    </h3>
     <p>
       If you're bored of grinding single player or want a new
       challenge, this is the place
       for you!
-      Play through automatically generated matches here 24/7 and
-      measure your skill up against others.
+      Play through randomly generated matches here anytime and
```

Comment:
Sorry to be arguing small points, but the match isn't generated "randomly", only the beatmaps are random. Also 'anytime' isn't an English word.